src/sys/mman.rs

```diff
@@ -184,6 +184,40 @@ libc_bitflags! {
         #[cfg(target_os = "openbsd")]
         #[cfg_attr(docsrs, doc(cfg(all())))]
         MAP_CONCEAL;
+        /// Pages aligned on 64kb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_64KB;
+        /// Pages aligned on 16mb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_16MB;
+        /// Pages aligned on 4gb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_4GB;
+        /// Pages aligned on 1tb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_1TB;
+        /// Pages aligned on 256tb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_256TB;
+        /// Pages aligned on 64pb
+        #[cfg(target_os = "netbsd")]
+        #[cfg_attr(docsrs, doc(cfg(all())))]
+        MAP_ALIGNMENT_64PB;
+        /// Right operand value for the page alignment bitshift calculation
+        /// FIXME: not present in libc for FreeBSD
```

Comment:
Why don't you go ahead and add defines for these on FreeBSD (and maybe other places) in libc before merging this PR into Nix?

Reply:
These 2 constants have been added in [libc#3326](https://github.com/rust-lang/libc/pull/3326)